themes/10up-theme/assets/css/frontend/global/typography.css

```diff
@@ -0,0 +1,38 @@
+/*
+ * Typography variables
+ */
+
+:root {
+	--ff-body: system-ui;
+	--ff-heading: system-ui;
+
+	/*
+	 * Font Sizes
+	 */
+	--fs-xxsmall: 13px;
```

Comment:
@AlejandroPerezMartin consider using --font-size

Reply:
@barneyjeffries both [color variables](https://github.com/10up/wp-scaffold/blob/trunk/themes/10up-theme/assets/css/frontend/global/colors.css) and [breakpoints](https://github.com/10up/wp-scaffold/blob/trunk/themes/10up-theme/assets/css/frontend/global/media-queries.css) are using short names, should I also update the naming there?